src/couch_jwt_auth.erl

```diff
@@ -58,6 +58,12 @@ decode(Token, Config) ->
       end, List), posix_time(calendar:universal_time()), Config)
   end.
 
+ensure_safe_token(Token, Config) ->
+  case couch_util:get_value(Token, Config) of
```

Comment:
How may tokens could be blacklisted? If you expect hundred/thousand values as real ones, better use `couch_config:get/3` here since ETS table behind is more suitable for random access than lists.